lib/LedgerNodeOperations.js

```diff
@@ -46,9 +48,41 @@ module.exports = class LedgerNodeOperations {
    * @return a Promise that resolves once the operation completes.
    */
   async validate(operation) {
+    if(!(operation && typeof operation === 'object')) {
+      throw new BedrockError('Operation must be an object.', 'TypeError');
+    }
+
+    _validateOperation(operation);
+
     const {ledgerNode} = this;
     const config = await ledgerNode.config.get();
     const validatorConfigs = config.operationValidator || [];
+
     return validator.validate(operation, validatorConfigs, {ledgerNode});
   }
 };
+
+function _validateOperation(operation) {
+  const context = operation['@context'];
+  _validateContext(context);
+  const validation = validate('webledger-validator.operation', operation);
+  if(!validation.valid) {
+    throw validation.error;
+  }
+}
+
+function _validateContext(context) {
+  const wlContextUrl = constants.WEB_LEDGER_CONTEXT_V1_URL;
+  if(!context || !(typeof context === 'string' || Array.isArray(context))) {
```

Comment:
It appears to me that this bit is handled by the JSON schema validation.  Is that right?

Reply:
Correct

Reply:
What JSON schema validator handles this? We need to make sure it doesn't allow a different order on the context(s).

Reply:
@dlongley the part that you're referring to is still in there.  The part we didn't need has already been removed and that's simple existence check and type checking which is already handled by JSON schema.

Reply:
Ok.